test/test_torch.py

```diff
@@ -11958,6 +11958,16 @@ def test_tensor_pow_tensor(values, torch_type, numpy_type):
         test_tensor_pow_tensor(floats, torch.float32, np.float32)
         test_tensor_pow_tensor(floats, torch.float64, np.float64)
 
+    @dtypes(torch.float)
+    def test_add_with_tail(self, device, dtype):
+        for tail_size in [1, 63, 67, 130]:
+            size = 4096 + tail_size
+            a = torch.randn(size, device=device, dtype=dtype)
+            b = torch.randn(size, device=device, dtype=dtype)
+            c = a + b
+            for x, y, z in zip(a.tolist(), b.tolist(), c.tolist()):
+                self.assertEqual(x + y, z)
+
```

Comment:
Sorry to ask this, but can we also have tests for weirdly aligned tensors here? I know you are testing `can_vectorize_up_to` separately, so here it will be to test end-to-end op flow?

Reply:
Sure, this sounds like a good idea

Reply:
Added in `test_logical_xor_with_nontrivial_alignment`.